packages/smooth_app/lib/main.dart

```diff
@@ -194,7 +194,8 @@ class _SmoothAppState extends State<SmoothApp> {
         themeProvider.colorTag,
       ),
       themeMode: themeProvider.currentThemeMode,
-      home: SmoothAppGetLanguage(appWidget),
+      // We need to wrap it inside a Scaffold to show error Snackbar's
+      home: Scaffold(body: SmoothAppGetLanguage(appWidget)),
```

Comment:
Not nice: it means embedded `Scaffold`s. Please do that another way.

Reply:
@monsieurtanuki  moved it lower, below the next and first Scaffold